libs/k6/build/index.js

```diff
@@ -0,0 +1,7 @@
+var E=Object.create;var c=Object.defineProperty;var y=Object.getOwnPropertyDescriptor;var R=Object.getOwnPropertyNames;var S=Object.getPrototypeOf,A=Object.prototype.hasOwnProperty;var i=(e,t)=>()=>(e&&(t=e(e=0)),t);var v=(e,t)=>()=>(t||e((t={exports:{}}).exports,t),t.exports),N=(e,t)=>{for(var s in t)c(e,s,{get:t[s],enumerable:!0})},u=(e,t,s,a)=>{if(t&&typeof t=="object"||typeof t=="function")for(let o of R(t))!A.call(e,o)&&o!==s&&c(e,o,{get:()=>t[o],enumerable:!(a=y(t,o))||a.enumerable});return e};var l=(e,t,s)=>(s=e!=null?E(S(e)):{},u(t||!e||!e.__esModule?c(s,"default",{value:e,enumerable:!0}):s,e)),T=e=>u(c({},"__esModule",{value:!0}),e);function p(){return{attachments:[{color:"#632eb8",blocks:[{type:"section",text:{type:"mrkdwn",text:"*|K6 Report Summary*"}},{type:"section",text:{type:"mrkdwn",text:""},accessory:{type:"image",image_url:"https://k6.io/images/landscape-icon.png",alt_text:"k6 thumbnail"}},{type:"divider"},{type:"actions",elements:[{type:"button",text:{type:"plain_text",text:"Grafana :grafana:",emoji:!0},value:"click_me_123",url:""}]}]}]}}var m=i(()=>{});var V,_=i(()=>{V={grafanaBaseUrl:__ENV.GRAFANA_BASE_URL??"https://altinn-grafana-test-b2b8dpdkcvfuhfd3.eno.grafana.azure.com",k6PrometheusDashboard:__ENV.K6_PROMETHEUS_DASHBOARD??"d/ccbb2351-2ae2-462f-ae0e-f2c893ad1028/k6-prometheus",datasource:__ENV.DATASOURCE??"k6tests-amw"}});var d=v(k=>{var h=l(require("k6/http"));m();_();function f(e){let t=e.metrics.http_reqs.values.count,s=e.metrics.http_reqs.values.rate.toFixed(2),a=e.metrics.http_req_duration.values.avg.toFixed(2),o=e.metrics.http_req_duration.values["p(95)"].toFixed(2),n=e.metrics.http_req_duration.values["p(99)"].toFixed(2);return`*Number of Requests:* ${t} reqs
```

Comment:
This will be removed in a later iteration (i.e. once we are ready to make a proper release)